proguard-tests/build.gradle.kts

```diff
@@ -1,3 +1,6 @@
+// This is always set to 'true' on Travis CI
+val inCiBuild = System.getenv("CI") == "true"
```

Comment:
I would check for nullity instead so we're resilient to possible changes: https://github.com/SUPERCILEX/Robot-Scouter/blob/2da4e7eed01da413eb151ce8cb09c75e33e7d53d/buildSrc/src/main/kotlin/com/supercilex/robotscouter/build/internal/Constants.kt#L8